extensions/Worker.Extensions.Kafka/src/Worker.Extensions.Kafka.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Kafka extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>3.3.2</VersionPrefix>
+    <VersionPrefix>3.3.3.1</VersionPrefix>
```

Comment:
`VersionPrefix` is supposed to have [3 parts](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/build/Common.props#L8), major, minor and patch.

Reply:
Suggest keeping the worker version number same as webjobs package version number.

Reply:
this is resolved now